src/screens/benefit/Details.tsx

```
@@ -212,17 +212,20 @@ const BenefitsDetails: React.FC = () => {
+				'application pending',
+				'submitted',
+			].includes(applicationStatus?.toLowerCase() || '');
+
+			const formData = isEditableStatus
 				? {
-					...(authUser || {}),
-					...(applicationData?.application_data || {}),
-					external_application_id: applicationData?.external_application_id,
-				  }
-				: authUser ?? undefined;
+						...(authUser || {}),
```

ESLint

[error] Expected indentation of 5 tabs but found 6.

(indent)